### PR TITLE
Fixed a bunch of links that used html instead of md as suffix.

### PR DESCRIPTION
**Pull request by stuclem**
Found a load of links that include `.html` in the Markdown, that should be `.md`. 

@anchal-agrawal can you please review?

### CODE REVIEW

File: docs/user_doc/vic_overview/introduction.md [hunk 1]
```diff
@@ -260,7 +260,7 @@ The following global permissions are unique for the cloud administrator role:
 
 vSphere Integrated Containers Registry is an enterprise-class registry server that you can use to store and distribute container images. vSphere Integrated Containers Registry allows DevOps administrators to organize image repositories in projects, and to set up role-based access control to those projects to define which users can access which repositories. vSphere Integrated Containers Registry also provides rule-based replication of images between registries, implements Docker Content Trust, and provides detailed logging for project and user auditing.
 
-For a more detailed overview of vSphere Integrated Containers Registry, see [Managing Images, Projects, and Users with vSphere Integrated Containers Registry](../vic_cloud_admin/using_registry.html) in *Configure and Manage vSphere Integrated Containers*.
+For a more detailed overview of vSphere Integrated Containers Registry, see [Managing Images, Projects, and Users with vSphere Integrated Containers Registry](../vic_cloud_admin/using_registry.md) in *Configure and Manage vSphere Integrated Containers*.
```
Comment (anchal-agrawal) [inline docs/user_doc/vic_overview/introduction.md]: This link 404s - is that expected?

Comment (stuclem) [reply docs/user_doc/vic_overview/introduction.md]: Ah, this is one where the whole topic has actually been deleted and its content migrated elsewhere. Well spotted!

Comment (stuclem) [reply docs/user_doc/vic_overview/introduction.md]: Removed the whole paragraph, as no longer necessary due to content reorg.

**Comment by stuclem** [comment]
Thanks @anchal-agrawal. Incidentally, I opened https://github.com/vmware/vic-product/issues/664 to make sure that we don't miss any more of these.

**Pull request merged**